rtt_tf/rtt_tf-component.cpp

```diff
@@ -294,6 +332,48 @@ namespace rtt_tf
     port_tf_static_out.write(msg_out);
   }
 
+  bool RTT_TF::subscribeTransfrom(const std::string& target, const std::string& source)
+  {
+    Logger::In(this->getName());
+    const std::pair<std::string, std::string> transfrom_pair(target, source);
+    const std::string name_transform = source + std::string("_") + target;
+    const std::string name_port = std::string("tracker_tf_") + name_transform;
+    if (ports_trackers.find(transfrom_pair) != ports_trackers.end()) {
+      RTT::log(RTT::Warning) << "The transfrom from " << source << " to " << target <<
+        " is already tracked." << RTT::endlog();
+      return false;
+    }
+    if (!canTransform(target, source)) {
```

Comment:
Just to be clear: subscribing even though the transform is not yet available is the intentional behavior, correct?

Reply:
Yes, the transform could be broadcasted later because the producing component hasn't get operational yet, but trying to read is safe. If no transform exists, then this port will not output data.